__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET /api/topics", () => {
+  test("200: Get successful status code", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+
+  test("200: Check returned result is an array", () => {
```

Comment:
We do not need this test because if we are using forEach on the one below it is implicit that it is an array 🙂